hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/table/HoodieTableSource.java

```diff
@@ -410,7 +410,7 @@ private List<MergeOnReadInputSplit> buildInputSplits() {
           return cdcInputFormat(rowType, requiredRowType, tableAvroSchema, rowDataType, result.getInputSplits());
         } else {
           return mergeOnReadInputFormat(rowType, requiredRowType, tableAvroSchema,
-              rowDataType, result.getInputSplits(), false);
+              rowDataType, result.getInputSplits(), true);
```

Comment:
Not sure we should do this, the flink batch operator can not handle the deletes correctly.